src/components/AboutUs/aboutUs.astro

```diff
@@ -0,0 +1,77 @@
+---
+import Layout from "../layouts/Layout.astro";
+import aboutUsImg from "/AboutUs/about-us.svg"
+import arrow from "/AboutUs/arrow-icon.svg"
+---
+
+<div class="w-full bg-[#0C1030] box-border flex justify-between flex-col md:flex-col xl:flex-row lg:flex-row">
+    <div class=" w-full flex align-center p-16 md,lg:p-20 md,lg,xl:w-1/2">
```

Comment:
commas arent used in tailwind
remove md,lg p-20 xl:w-1/2 
this doesnt work like this